src/app.rs

```diff
@@ -128,6 +131,93 @@ impl App{
         self.zoom += self.zoom_factor;
     }
 
+    pub fn show_help<B>(&mut self, f: &mut Frame<B>)
+    where
+        B: Backend,
+    {
+        // Text
+        let key_bindings_raw = vec![
```

Comment:
its missing  + to increase zoom and -to decrease zoom, which is implemented the wrong way around (I will create a pr)